src/ADAL.PCL/AcquireTokenHandlerBase.cs

```diff
@@ -304,9 +323,8 @@ protected virtual void PostTokenRequest(AuthenticationResultEx result)
 
         private async Task<AuthenticationResultEx> SendHttpMessageAsync(IRequestParameters requestParameters)
         {
-            var client = new AdalHttpClient(this.Authenticator.TokenUri, this.CallState) { Client = { BodyParameters = requestParameters } };
+            client = new AdalHttpClient(this.Authenticator.TokenUri, this.CallState) { Client = { BodyParameters = requestParameters } };
```

Comment:
this should be automatically reformatted. don't put linebreaks manually. use VS/Resharper formatting.